client/src/components/main/notification/NotificationCenter.js

```diff
@@ -56,6 +57,19 @@ export default class NotificationCenter extends React.Component {
       }
       return 0;
     });
+    const {systemMaintenanceMode, systemMaintenanceModeBanner} = this.props.preferences;
+    if (systemMaintenanceMode && systemMaintenanceModeBanner) {
+      sortedNotifications.unshift({
+        blocking: false,
+        body: systemMaintenanceModeBanner,
+        createdDate: '',
+        notificationId: -1,
```

Comment:
Let's declare `PredefinedNotifications` outside of the `NotificationCenter` component and use it here:
```js
const PredefinedNotifications = {
  MaintenanceMode: -1
};
...
notificationId: PredefinedNotifications.MaintenanceMode,
...
```